bigquery/deploy/README.md

```diff
@@ -0,0 +1,71 @@
+## Running on GKE
+
+* Run Elasticsearch on GKE
+  * Clone `kubernetes-elasticsearch-cluster` repo.
+    ```
+    git clone https://github.com/pires/kubernetes-elasticsearch-cluster.git
+    cd kubernetes-elasticsearch-cluster
+    ```
+  * Set up [Internal Load Balancer](https://cloud.google.com/kubernetes-engine/docs/how-to/internal-load-balancing).
+  Note: This is not needed for indexing. This will be needed for having API
+  server on App Engine Flex talk to Elasticsearch in GKE; might as well set up
+  now. Change `es-svc.yaml` to:
+    ```
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: elasticsearch
+      labels:
+        component: elasticsearch
+        role: client
+      annotations:
+        cloud.google.com/load-balancer-type: "Internal"
+    spec:
+      selector:
+        component: elasticsearch
+        role: client
+      ports:
+      - name: http
+        port: 9200
+      type: LoadBalancer
+    ```
+  * Create cluster
+    * Go to https://console.cloud.google.com/kubernetes/list and click `Create Cluster`
+    * Change name to `elasticsearch-cluster`
+    * Change `Machine type` to `4 vCPUs`. (Otherwise will get Insufficient CPU error.)
```

Comment:
2 vCPUs per node is sufficient here

Reply:
Really we just need `numNodes * (vCPU per node - 1) > totalNumServices` which in this case means we'd need 4 nodes using 2 vCPU each. This is just a matter of preference but the reason why I prefer to set it up this way is because it ensures that as we scale this up, we never run into situations where a node is down containing all of our data pods, which would lead to a total outage. This is actually more of a workaround to setting up true [pod anti-affinity](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#inter-pod-affinity-and-anti-affinity-beta-feature) but this is more of a scalability concern to keep in mind for the future more so than something we need to take care of now.